### ISSUE TITLE
Any maintenance updates?

### ISSUE THREAD

**Issue by FireEmerald** [puikinsh/Dazzling#73]
@puikinsh Are there any plans for maintenance updates for this great free theme from your side in the **near** future?

It's because the theme requires some more love after the latest WordPress update to 5.6 and the deprecation of jQuery.

**Comment by mario-tux** [comment]
Yes, please.

**Comment by maciejbaur** [comment]
It will be great to make an update because now we have to use jQuery Migrate plugin to solve that issue.

**Comment by puikinsh** [comment]
Yes — the theme has had a substantial maintenance pass. Current release is **2.2.5**.

What has shipped since you asked:

- **Security.** The Customizer's colour sanitiser returned its input unchanged when validation failed, so arbitrary text could be stored through a colour setting and printed into the inline `<style>` block on every page. The custom CSS option ran through `html_entity_decode()`, which turned an escaped `</style><script>` back into live markup. The per-post layout metabox saved whatever was submitted and `header.php` printed it unescaped into a class attribute. All fixed and tested against stored attack values.
- **Bootstrap 3.4.1**, which is the release that fixed CVE-2019-8331. The theme was on 3.3.6.
- **Font Awesome 7.3.1**, self-hosted and subsetted, replacing 4.4.0.
- **Internet Explorer support dropped** — html5shiv, Respond.js and the `X-UA-Compatible` meta are gone.
- **The featured slider fixed** (#36, #59, #61, #71 — all closed today).
- Verified on WordPress 7.1 / PHP 8.5 with no theme errors, and it passes the current Theme Check with no required items.

On jQuery specifically: the theme still depends on it, because it is on Bootstrap 3, whose JavaScript requires jQuery. Moving off that means moving off Bootstrap 3, which would change the markup every child theme is built against, so it has not been done. jQuery ships with WordPress and the theme works with the version WordPress 7.x provides.
